src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -3,7 +3,15 @@
 import awkward as ak
 
 
-def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
+def from_rdataframe(
+    rdf,
+    columns,
+    offsets_type="int64_t",
+    keep_order=False,
+    *,
+    highlevel=True,
+    behavior=None,
+):
```

Comment:
I think it would be more natural to put the separator (`*`) between non-keyword-only and keyword-only arguments directly after `columns`.

Generally speaking, any boolean parameters should be keyword-only because a bare word "`False`" or "`True`" has little information, but also any rarely used arguments, which definitely includes `offsets_type`.

```suggestion
def from_rdataframe(
    rdf,
    columns,
    *,
    keep_order=False,
    offsets_type="int64_t",
    highlevel=True,
    behavior=None,
):
```